ipc/shmem_ipc/Makefile

```diff
@@ -1,30 +1,30 @@
 CC=gcc
-CFLAGS= -O3 -Wall -Wextra -Wno-unused-result -Wno-unused-variable
+CFLAGS= -O3 -Wall -Wextra -Wno-unused-result -Wno-unused-variable -lpthread
 
 SYMLIB_DIR=../../../Symlib
 SYMLIB_LINK=-lSym
 SYMLIB_INCLUDE=$(SYMLIB_DIR)/include
 SYMLIB_LIB_DIR=$(SYMLIB_DIR)/dynam_build
 
-all: independent_client independent_client_elevated client server server_elevated
+all: client_elevated client server server_elevated serverkill
```

Comment:
Just to clarify, why is the client also being elevated?

Reply:
If that's your independent client, let's keep the naming consistent and call it independent_client, what do you think?

Reply:
the client will take the command line argument to determine whether it does the test independently or connects to the server. See more detail in the README. Either making separate binaries or consuming the command line argument works fine. I'm okay with either way.